recipes/amplify/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/
+cp -r src/. $PREFIX/bin/
```

Comment:
The bin folder should only contain the actual executables. I suggest to copy the contents of src into `$PREFIX/share/amplify`, and put a script like

```
#!/bin/bash

$PREFIX/share/amplify/amplify.py $@
```

into the bin folder. You will need to generate this via echo, such that $PREFIX ends up to be the correct path in the script.

Reply:
Thanks for the suggestion! We have made changes to the scripts.